ci/common.cfg

```diff
@@ -14,6 +14,9 @@
 
 # Format: //devtools/kokoro/config/proto/build.proto
 
+# Build timeout set for 10 hours
+timeout_mins: 600
```

Comment:
Any build that takes this long gives me a moment's pause, and makes me ask if we're using the right architecture.  Approving to unblock things, but please file a follow up issue to dig into what's happening here.

Reply:
I suppose this is because of the nature of the "FORCE_GENERATE_ALL" build which inevitably takes too long to flush out all the docs. Will check in with Tyler to see what we can do for long builds or possibly see where we can cut time.